orchagent/portsorch.cpp

```diff
@@ -6883,3 +7132,115 @@ std::unordered_set<std::string> PortsOrch::generateCounterStats(const string& ty
     }
     return counter_stats;
 }
+
+void PortsOrch::updatePortStateAutoNeg(const Port &port)
```

Comment:
The function name is misleading. Should it be updatePortRemoveAdvSpeeds?

Reply:
This is a placeholder for the operational AN state, not specific to remote speed only, there are more upcoming in the PRs in the furture (e.g. tx/rx pause, FEC ....)